proposals/lip-0051.md

```diff
@@ -1879,9 +1874,9 @@ def transferCrossChain(
             raise Exception("Insufficient sender available balance")
 
     # Transfer is only possible for tokens native to either sending, receiving or mainchain.
-    if tokenChainID not in [OWN_CHAIN_ID, receivingChainID, getMainchainID()]:
+    if tokenChainID not in [OWN_CHAIN_ID, receivingChainID]:
```

Comment:
The comment in the line above must be updated as well.

Reply:
good catch, fixed.